src/Symfony/Component/Security/Csrf/TokenStorage/AbstractTokenStorageProxy.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Csrf\TokenStorage;
+
+/**
+ * Forwards calls to another TokenStorageInterface.
+ *
+ * @author Oliver Hoff <oliver@hofff.com>
+ */
+abstract class AbstractTokenStorageProxy implements TokenStorageInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @see \Symfony\Component\Security\Csrf\TokenStorage\TokenStorageInterface::getToken()
```

Comment:
all those `@see` should be removed, they do not add anything useful (the class implements `TokenStorageInterface`).